plugins/TwoFactorAuth/Controller.php

```diff
@@ -148,6 +152,13 @@ public function disableTwoFactorAuth()
             $this->twoFa->disable2FAforUser(Piwik::getCurrentUserLogin());
             $this->passwordVerify->forgetVerifiedPassword();
 
+            $container = StaticContainer::getContainer();
+            $email = $container->make(TwoFactorAuthDisabledEmail::class, array(
+                'login' => Piwik::getCurrentUserLogin(),
+                'emailAddress' => Piwik::getCurrentUserEmail()
+            ));
+            $email->send();
```

Comment:
Just for completeness: If sending the mail fails for any reason this might throw an exception, which is currently uncaught.  It might then show an error message and the user might not be sure if the action was actually performed or not. Not sure if we should maybe better ignore possible exceptions in this case. Same applies for the other mails